src/Heading.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <header className="App-header">
+      <img className="logo" alt="Heading logo" src="../public/Hotel1.png" />
```

Comment:
create-react-app does some magic with the `public` directory- the correct `src` for this is just `/Hotel1.png`